### PR TITLE
D8CORE-2349: updating person list view to a single link

### PR DESCRIPTION
**Pull request by cjwest**
# READY FOR REVIEW

# Summary
- Changes to the two person views are to wrap the photo and the title (person's name) in a single <a>

# Review By (Date)
- v1.5
Addresses accessibility issues

# Criticality
- How critical is this PR on a 1-10 scale? 1 - Addresses accessibility issues (https://stanfordits.atlassian.net/browse/D8CORE-1705).
- It affects every site using D8Core

# Review Tasks

There are two views involved.

## Setup tasks and/or behavior to test

1. Check out this branch
2. Navigate to the person list page, i.e. `/people`
1. Verify that for each profile, both the title and the image are linked in the same `<a>`
1. Navigate to a taxonomy term page, 1.e. `/people/faculty`
1. Verify that for each profile, both the title and the image are linked in the same `<a>`

## Front End Validation
- [ √] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ √] Cross-browser testing has been performed? - Firefox, Chrome, IE11
- [ ] Automated accessibility scans performed?
- [√ ] Manual accessibility tests performed? - tab
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s): https://stanfordits.atlassian.net/browse/D8CORE-2349
- Other PRs: https://github.com/SU-SWS/stanford_profile/pull/281
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: config/install/views.view.stanford_person.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+uuid: cf1fc1a9-9b4f-4128-b71e-1acc0d673cbd
```
Comment (sherakama) [inline config/install/views.view.stanford_person.yml:1]: No uuids in module files.

Comment (cjwest) [reply config/install/views.view.stanford_person.yml:1]: @sherakama,
How do I best address this issue? Just delete the line, or did I miss something in the development process?

Comment (sherakama) [reply config/install/views.view.stanford_person.yml:1]: Just delete it.

**Comment by cjwest** [comment]
Got it! Thanks, @sherakama !

**Review by sherakama — APPROVED** [review]
GTG on green lights.

**Pull request merged**